index.html

```diff
@@ -14,72 +14,74 @@
           /
         </button>
       </div>
       <div class="calculator-row">
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="7">
           7
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="8">
           8
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="9">
           9
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="X">
           X
         </button>
       </div>
       <div class="calculator-row">
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="4">
           4
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="5">
           5
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="6">
           6
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="-">
           -
         </button>
       </div>
       <div class="calculator-row">
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="1">
           1
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="2">
           2
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="3">
           3
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="+">
           +
         </button>
       </div>
       <div class="calculator-row">
-        <button class="calculator-button calculator-container calculator-zero">
+        <button class="calculator-button calculator-container calculator-zero" value="0">
           0
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value=".">
           .
         </button>
-        <button class="calculator-button calculator-container">
+        <button class="calculator-button calculator-container" value="=">
           =
         </button>
       </div>
     </div>
   </div>
 </body>
+<script src="./Calculator.js"></script>
```

Comment:
Lets try and keep all of our code in one script file. `Calculator.js` is a decent name. 


You shouldn't need a controller if you build all of that into your `Calculator` object.